Classes/Private/Manager/Database/OCTDBManager.m

```diff
@@ -257,6 +257,27 @@ - (RLMResults *)callsWithChat:(OCTDBChat *)chat
     return results;
 }
 
+- (OCTDBCall *)getOrCreateCallWithFriendNumber:(NSInteger)friendNumber
+{
+    OCTDBChat *chat = [self getOrCreateChatWithFriendNumber:friendNumber];
+    __block OCTDBCall *call = [[self callsWithChat:chat] lastObject];
```

Comment:
This `lastObject` is a little bit confusing. Why last call is right one?

AFAIU one chat can have only one call? Then maybe you can change `callsWithChat` to `callWithChat`?